SlothfulSheep/bangazon-api-sprint2
# User is able to add payment option for a customer

# Description

When an active customer is selected, the user is is able to then add a payment option for that customer.

Fixes #3 

## Type of change


# How Has This Been Tested?

In terminal, run
```node db/build_table.js```

Make sure all tests are passing
```npm test```

There should be 9 tests passing for payment types.

Run ```npm start``` to ensure the option to add payment types for a customer is functioning with CLI
- select option 2 to activate a customer
- select option 3 to add payment type for that customer
- follow the prompts to enter payment type and account number
- check the table in db browser to see if the payment option has been added for that customer


# Checklist:
